frontend/src/views/Resources.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-container>
+    <h1>Heading 1</h1>
+    <h2>Heading 2</h2>
+    <h3>Heading 3</h3>
+    <h4>Heading 4</h4>
+    <p>Paragraph</p>
+    <!-- <v-row>
+      <v-col class="pa-4" cols='6'>
+        Some text
+      </v-col>
+      <v-col class="pa-4" cols='6'>
+        Some other text
+      </v-col>
+    </v-row> -->
+    <div class="flex-grid">
+      <div class="col col--md-9">1</div>
+      <div class="col col--md-3">2</div>
+      <div class="col col--md-3">3</div>
+      <div class="col col--md-9">4</div>
+    </div>
+  </v-container>
+</template>
+
+<script>
+</script>
+
+<style lang="scss" scoped>
+.col {
+  background-color: $primary-color;
```

Comment:
This line is making the frontend fail its compilation.
@glebme When I comment it out, it works fine.

Reply:
Hmm, replace that line with a solid colour say #000000 and see if the columns in the Resources.vue page are still there. If they aren't then there's something wrong with the loader, else there's something wrong with the imports. Get back to me when you have the time.

Reply:
This is what I see when I change it to #000000.
![image](https://user-images.githubusercontent.com/47744329/90359336-03b5fd80-e09c-11ea-9eb7-dd1aa681e5df.png)

Reply:
This was just a Windows problem. The problem was that I ran `yarn install` on WSL rather than on Windows, so Docker (Windows) couldn't read the binaries created (since they are OS dependent).